src/annotator/integrations/pdf-metadata.ts

```diff
@@ -49,6 +49,23 @@ function pdfViewerInitialized(app: PDFViewerApplication): Promise<void> {
   }
 }
 
+/**
+ * Wait for PDF to be downloaded.
+ *
+ * For PDF.js versions older than v4.5, we rely on
+ * `PDFViewerApplication.downloadComplete`.
+ * For newer PDF.js versions we wait for
+ * `PDFViewerApplication.pdfDocument.getDownloadInfo()` to resolve.
+ */
+async function isPDFDownloaded(app: PDFViewerApplication): Promise<boolean> {
```

Comment:
This function has type `Promise<boolean>` but it only ever returns true. An API like `whenPDFDownloaded(app): Promise<void>` would probably give a better cue to readers about how it behaves.